tests/functional/configuration/config.py

```diff
@@ -62,4 +60,13 @@
 )
 
 # OAUTH
-OAUTH_PROXY = ENV['oauth-proxy']
+OAUTH_PROXY = ENV['oauth_proxy']
+
+SPINE_HOSTNAME = (
+    # This value is the url returned in the patients response payload which reflects a spine environment.
+    # internal-qa environment points to spine int environment.
+    "https://veit07.api.service.nhs.uk" if ENVIRONMENT == "internal-dev" else "https://int.api.service.nhs.uk"
```

Comment:
Would it make sense for the VEIT07 and INT paths to be under environment variables?

Reply:
I agree. Will add this to my 'to do' list. Thanks.

Reply:
Done.